modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -14,7 +14,15 @@ Working with {prod-short} in your browser involves multiple URLs:
 * [.underline]#The URLs for starting a new workspace#
 * The URLs of your workspaces in use
 
-With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. This way, you can clone a Git repository that is hosted on GitHub, a GitLab instance, or a Bitbucket server.
+With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. You can use either an HTTPS or SSH URL to start a new workspace with a clone of a Git repository.
+
+[IMPORTANT]
+====
+//TODO: point to the dashboard SSH procedure when https://github.com/eclipse/che/issues/22111 is done.
+* If an SSH URL is used to start a new workspace, the SSH key must be propagated, see link:https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#configuring-devworkspaces-to-use-ssh-keys-for-git-operations[Configuring DevWorkspaces to use SSH keys for Git operations].
+* If the SSH URL points to a private repository, to be able to fetch the `devfile.yaml` content, an access token must be applied by accepting an SCM authentication page or xref:using-a-git-provider-access-token.adoc[Personal Access Token] procedures.
```

Comment:
for private repos both PAT and OAuth should work, right?

Reply:
yes